Docs: Fix ftype list in sefcontext module

major opened the pull request:
##### SUMMARY
The file type list in the `sefcontext` module documentation does not display
the list of file types. Adjust the YAML formatting to ensure the list is
displayed.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
sefcontext module

##### ANSIBLE VERSION
```
ansible 2.6.5
  config file = None
  configured module search path = [u'/home/major/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /home/major/.pyenv/versions/2.7.14/envs/venv-2.7.14/lib/python2.7/site-packages/ansible
  executable location = /home/major/.pyenv/versions/venv-2.7.14/bin/ansible
  python version = 2.7.14 (default, Mar  7 2018, 08:13:01) [GCC 4.2.1 Compatible Clang 5.0.1 (tags/RELEASE_501/final)]
```

dagwieers commented on lib/ansible/modules/system/sefcontext.py:
```diff
@@ -27,16 +27,15 @@
-      C(l) for symbolic links,
-      C(p) for named pipes,
-      C(s) for socket files.
+    - "The following file type options are available:"
```
We tend to use the first line in the description to explain what the option is for.
Granted "File type." was not really helpful, but a short sentence would be nice.

Rationale: In the future we want to highlight the first line from the description in the module docs.

major replied:
That makes sense to me. I'll go tinker with that a bit more and put up a new revision.

major committed:
Docs: Fix ftype list in sefcontext module

The file type list in the `sefcontext` module documentation does not display
the list of file types. Adjust the YAML formatting to ensure the list is
displayed.

Signed-off-by: Major Hayden <major@redhat.com>

acozine commented:
Nice enhancement, thanks @major.
